test/functional/change_stream.test.js

```diff
@@ -2018,15 +2028,14 @@ describe('Change Streams', function() {
             .then(() => changeStream.next())
             .then(() => changeStream.next())
             .then(() => {
-              const nextP = changeStream.next();
-
-              return changeStream.close().then(() => nextP);
+              const nextP = () => changeStream.next();
```

Comment:
I had to change nextP into a function to prevent the `.next` from returning successfully before the `close` on the next line.